examples/module_mock/__tests__/partial_mock.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+/**
+ * This file illustrates how to do a partial mock where a subset
+ * of a module's exports have been mocked and the rest
+ * keep their actual implementation.
+ */
+import DefaultExport, {apple, strawberry} from '../fruit';
+
+jest.mock('../fruit', () => {
+  const originalModule = require.requireActual('../fruit');
+
+  const mockedModule = jest.genMockFromModule('../fruit');
+
+  const module = Object.assign(mockedModule, originalModule);
+
+  // Module the default export and named export 'apple'
+  module.default = jest.fn(() => 'mocked fruit');
+  module.apple = 'mocked apple';
+  return module;
```

Comment:
Initially I also thought this was the way to do it but changing the code this way generates an error:

<img width="542" alt="screen shot 2017-08-06 at 08 45 00" src="https://user-images.githubusercontent.com/19558321/29001223-9272a4ec-7a83-11e7-89fb-da42b2ea2316.png">

The default export cannot be mocked this way unfortunately:( Do you have any idea why this is happening?